reorder_python_imports.py

```diff
@@ -522,6 +522,9 @@ def _report_diff(contents: str, new_contents: str, filename: str) -> None:
     ),
     ('py37', ('generator_stop',)),
 )
+MYPY_EXTENSIONS_IMPORTS: Tuple[Tuple[str, Tuple[str, ...]]] = (
+    ('py38', ('NoReturn', 'TypedDict')),
```

Comment:
NoReturn was in python3.7 too and 3.6.something

Reply:
TIL. Sounds like I should rewrite `NoReturn` in `--py37-plus` then, or should I assume `--py36-plus` points to the latest py36?

Reply:
idk :S

should we do typing-extensions too while we're at it?

Reply:
For the first point, `--py37-plus` seems safest. For the second, sure, just need to read the source/changeloges and figure out which imports got promoted in which version